modules/machineset-yaml-osp.adoc

```diff
@@ -66,6 +66,7 @@ spec:
           trunk: true
           userDataSecret:
             name: <node_role>-user-data <2>
+          availabilityZone: <optional_list_of_openstack_availability_zones>
```

Comment:
Sorry I only now see that this is marked as `optional_list_of_openstack_availability_zones`, however a MachineSet is AZ-specific (if you want to have machines on 3 AZs, you get to create 3 MachineSets, one per zone).

So this would be `optional_openstack_availability_zone`

Reply:
Thanks!